apps/src/studio/blocks.js

```diff
@@ -1809,6 +1809,45 @@ exports.install = function (blockly, blockInstallOptions) {
     });
   };
 
+  blockly.Blocks.studio_whenSpriteAndGroupCollide = {
```

Comment:
Should `<block type="studio_whenSpriteAndGroupCollide" />` be added to [`Studio#common_blocks`](https://github.com/code-dot-org/code-dot-org/blob/fa8b6a956ac1d07c834d8b75f54d2a370d622803/dashboard/app/models/studio.rb#L120)?